include/deal.II/base/geometry_info.h

```diff
@@ -3051,10 +3050,1660 @@ unsigned int
 GeometryInfo<4>::line_to_cell_vertices (const unsigned int,
                                         const unsigned int)
 {
-  Assert(false, ExcNotImplemented());
+  AssertThrow(false, ExcNotImplemented());
```

Comment:
Not that it matters, but why?

Reply:
Some leftovers when trying to make the class `constexpr`. For conformity, I changed all the `AssertThrow (false, ExcNotImplemented())` to `Assert (false, ExcNotImplemented())` now.